2021-04-08 08:47:27,024 INFO    MainThread:1183764 [wandb_init.py:_log_setup():292] Logging user logs to /home/menglh/rlproj/tmp/on-policy/onpolicy/scripts/results/StarCraft2/corridor/mappo/mlp/wandb/run-20210408_084727-2ewfhzqx/logs/debug.log
2021-04-08 08:47:27,025 INFO    MainThread:1183764 [wandb_init.py:_log_setup():293] Logging internal logs to /home/menglh/rlproj/tmp/on-policy/onpolicy/scripts/results/StarCraft2/corridor/mappo/mlp/wandb/run-20210408_084727-2ewfhzqx/logs/debug-internal.log
2021-04-08 08:47:27,025 INFO    MainThread:1183764 [wandb_setup.py:_flush():68] setting env: {}
2021-04-08 08:47:27,025 INFO    MainThread:1183764 [wandb_setup.py:_flush():68] setting user settings: {}
2021-04-08 08:47:27,025 INFO    MainThread:1183764 [wandb_setup.py:_flush():68] multiprocessing start_methods=fork,spawn,forkserver
2021-04-08 08:47:29,171 INFO    MainThread:1183764 [wandb_run.py:_console_start():1265] atexit reg
2021-04-08 08:47:29,173 INFO    MainThread:1183764 [wandb_run.py:_redirect():1135] redirect: SettingsConsole.REDIRECT
2021-04-08 08:47:29,174 INFO    MainThread:1183764 [wandb_run.py:_redirect():1138] Redirecting console.
2021-04-08 08:47:29,174 INFO    MainThread:1183764 [redirect.py:install():196] install start
2021-04-08 08:47:29,175 INFO    MainThread:1183764 [redirect.py:install():211] install stop
2021-04-08 08:47:29,175 INFO    MainThread:1183764 [redirect.py:install():196] install start
2021-04-08 08:47:29,176 INFO    MainThread:1183764 [redirect.py:install():211] install stop
2021-04-08 08:47:29,176 INFO    MainThread:1183764 [wandb_run.py:_redirect():1182] Redirects installed.
2021-04-08 08:47:32,632 INFO    MainThread:1183764 [wandb_run.py:_atexit_cleanup():1238] got exitcode: 255
2021-04-08 08:47:32,634 INFO    MainThread:1183764 [wandb_run.py:_restore():1210] restore
2021-04-08 08:47:32,635 ERROR   stderr    :1183764 [redirect.py:_pipe_relay():129] problem in pipe relay
Traceback (most recent call last):
  File "/home/menglh/miniconda3/envs/ob_mappo/lib/python3.6/site-packages/wandb/lib/redirect.py", line 127, in _pipe_relay
    cb(name, data)
  File "/home/menglh/miniconda3/envs/ob_mappo/lib/python3.6/site-packages/wandb/sdk/wandb_run.py", line 634, in _console_callback
    self._backend.interface.publish_output(name, data)
  File "/home/menglh/miniconda3/envs/ob_mappo/lib/python3.6/site-packages/wandb/interface/interface.py", line 146, in publish_output
    self._publish_output(o)
  File "/home/menglh/miniconda3/envs/ob_mappo/lib/python3.6/site-packages/wandb/interface/interface.py", line 151, in _publish_output
    self._publish(rec)
  File "/home/menglh/miniconda3/envs/ob_mappo/lib/python3.6/site-packages/wandb/interface/interface.py", line 429, in _publish
    raise Exception("The wandb backend process has shutdown")
Exception: The wandb backend process has shutdown
2021-04-08 08:47:32,635 INFO    MainThread:1183764 [redirect.py:uninstall():215] uninstall start
2021-04-08 08:47:32,637 INFO    MainThread:1183764 [redirect.py:_stop():264] _stop: stdout
2021-04-08 08:47:32,637 INFO    MainThread:1183764 [redirect.py:_stop():270] _stop closed: stdout
2021-04-08 08:47:32,637 INFO    stdout    :1183764 [redirect.py:_pipe_relay():119] relay done saw last write: stdout
2021-04-08 08:47:32,638 INFO    stdout    :1183764 [redirect.py:_pipe_relay():135] relay done done: stdout
2021-04-08 08:47:32,638 INFO    MainThread:1183764 [redirect.py:_stop():276] _stop joined: stdout
2021-04-08 08:47:32,638 INFO    MainThread:1183764 [redirect.py:_stop():278] _stop rd closed: stdout
2021-04-08 08:47:32,638 INFO    MainThread:1183764 [redirect.py:uninstall():219] uninstall done
2021-04-08 08:47:32,638 INFO    MainThread:1183764 [redirect.py:uninstall():215] uninstall start
2021-04-08 08:47:32,638 INFO    MainThread:1183764 [redirect.py:_stop():264] _stop: stderr
2021-04-08 08:47:32,638 INFO    MainThread:1183764 [redirect.py:_stop():270] _stop closed: stderr
2021-04-08 08:47:32,638 INFO    stderr    :1183764 [redirect.py:_pipe_relay():119] relay done saw last write: stderr
2021-04-08 08:47:32,638 INFO    stderr    :1183764 [redirect.py:_pipe_relay():135] relay done done: stderr
2021-04-08 08:47:32,638 INFO    MainThread:1183764 [redirect.py:_stop():276] _stop joined: stderr
2021-04-08 08:47:32,638 INFO    MainThread:1183764 [redirect.py:_stop():278] _stop rd closed: stderr
2021-04-08 08:47:32,638 INFO    MainThread:1183764 [redirect.py:uninstall():219] uninstall done
2021-04-08 08:47:32,639 INFO    MainThread:1183764 [wandb_run.py:_restore():1210] restore
2021-04-08 08:47:32,834 INFO    MainThread:1183764 [internal.py:handle_exit():137] Internal process exited
